IPython/core/interactiveshell.py

```diff
@@ -479,6 +479,10 @@ def input_splitter(self):
         """
     ).tag(config=True)
 
+    warn_venv = Bool(
+        True, help="Warn on startup if running inside a virtual environment."
```

Comment:
That's not what happens. The warning is shown if virtualenv is active but IPython runs _outside_ of it.

Reply:
That's what I meant – I agree the wording is ambiguous. What about:
"Warn if a virtual environment is active, and IPython is installed outside of it."

Reply:
Oh, I get it now. Could you please rephrase this help message so that it is clear?

This would confuse people who a re looking at it from a PoV of trying to run outside of a virtualenv.

Reply:
Is this OK?

"Warn if running in a virtual environment which does not have IPython installed (so IPython from the global environment is used)."

Reply:
I ended up including this text:

'Warn if running in a virtual environment with no IPython installed (so IPython from the global environment is used).'

I also added the what's new entry.